backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/invoice/isCompleteInvoice.feature

```diff
@@ -0,0 +1,120 @@
+@from:cucumber
+@dev:runThisOne
+Feature: completeInvoice option when processing invoice candidates
+
+  Background:
+    Given the existing user with login 'metasfresh' receives a random a API token for the existing role with name 'WebUI'
```

Comment:
why do you need to have an API-token?

Reply:
without it AD_User_ID is missing in ctx and it can't be invoiced